src/Components/LikeBtn/LikeBtn.js

```diff
@@ -1,14 +1,35 @@
 import React from "react";
+import { useUserActions } from "../utils/useUserActions";
 
 export const LikeBtn = ({ prod }) => {
+  const {
+    isPresentInWishlist,
+    addProductInWishlist,
+    removeProductFromWishlist,
```

Comment:
This can also be shortened to `addToWishlist` `removeFromWishlist` 

There is no need to explicitly mention products and we are only working with products